packages/wallets/src/AppContent.scss

```diff
@@ -4,12 +4,12 @@
     align-items: center;
     gap: 24px;
     width: 100%;
-    min-height: 100%;
+    min-height: calc(100vh - 8.4rem); // 100vh - (4.8rem header + 3.6rem footer)
```

Comment:
Why can't we use 100%?

Reply:
It doesnt take the whole page and only follows the height of the content